test/e2e/cra-5/package.json

```diff
@@ -5,7 +5,11 @@
   "dependencies": {
     "@thirdweb-dev/react": "workspace:*",
     "@thirdweb-dev/sdk": "workspace:*",
+    "browserify-fs": "^1.0.0",
     "ethers": "^5.7.2",
+    "fs": "npm:browserify-fs",
```

Comment:
we might need to expose `DeviceWallet.node()` in a separate export i think. safer than having node stuff in there that might break browser

Reply:
done, moved it to separate export and reverted the changes done in cra-5 package.json

New API: 

```ts
import { DeviceWalletNode } from '@thirdweb-dev/wallets/evm/wallets/device-walllet-node';

// use default file path for saving the data ("./wallet.json")
const deviceWallet = new DeviceWalletNode();

// OR 

// specify a file path for saving the data
const deviceWallet = new DeviceWalletNode({
  storageJsonFile: './xyz.json',
});
```